pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>io.airlift</groupId>
         <artifactId>airbase</artifactId>
-        <version>64</version>
+        <version>64-ERIC</version>
```

Comment:
This commit adds `presto-connector-tests` maven module but not list that in `<modules>` section of top-level `pom.xml`.